internal/provider/resource_ably_rule_amqp_external_test.go

```diff
@@ -98,6 +101,7 @@ func testAccAblyRuleAmqpExternalConfig(
 	sourceType string,
 	targetUrl string,
 	targetRoutingKey string,
+	targetExchange string,
 	targetManditoryRoute bool,
```

Comment:
I realise this was there before, but typo (Manditory -> Mandatory)

Reply:
I shall fix this. Thanks for flagging it.

Reply:
Fixed in https://github.com/ably/terraform-provider-ably/pull/170/commits/bd73030e798fece355bd3de66676279b2623e3e8